AutomatedLab/AutomatedLabAzure.psm1

```diff
@@ -74,9 +74,9 @@ function Add-LabAzureSubscription
 
     #This needs to be loaded manually to import the required DLLs
     $minimumAzureModuleVersion = $MyInvocation.MyCommand.Module.PrivateData.MinimumAzureModuleVersion
-    if (-not (Get-Module -Name Azure -ListAvailable | Where-Object Version -ge $minimumAzureModuleVersion))
+    if (-not (Get-Module -Name AzureRM -ListAvailable | Where-Object Version -ge $minimumAzureModuleVersion))
     {
-        throw "The Azure PowerShell module version $($minimumAzureModuleVersion) or greater is not available. Please download it from 'http://azure.microsoft.com/en-us/downloads/'"
+        throw "The Azure PowerShell module version $($minimumAzureModuleVersion) or greater is not available. Please install it: Install-Module AzureRM -Force"
```

Comment:
IMHO this would be a change that should not be done by AL. AL does not have many requirements but these ones should be installed manually.

Unfortunately, the platform installer installes the modules in a different way than Install-Module does. We need it the way Install-Module does it.